include/QQ_MACROS.h

```diff
@@ -38,7 +38,11 @@
 	public:    const type & getf() { return val; }\
 
 
+<<<<<<< a5433d807b1c19427640022fd1f38dedc6641a5f
```

Comment:
If I get any odd messages I often search a file, or the whole code, for `<<<`.
It's a marker used by git on merge actions. You can see what files are failing with the command `git status`.